components/ingest-service/config/old_job_config.go

```diff
@@ -0,0 +1,96 @@
+package config
+
+import (
+	"io/ioutil"
+	"os"
+
+	toml "github.com/pelletier/go-toml"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+// Enum jobs
+const (
+	DeleteNodes int = iota
+	NodesMissing
+	MissingNodesForDeletion
+)
+
+// List of jobs
+var JobList = map[int]string{
+	DeleteNodes:             "delete_nodes",
+	NodesMissing:            "missing_nodes",
+	MissingNodesForDeletion: "missing_nodes_for_deletion",
+}
+
+// JobConfig is the config for a job
+type JobConfig struct {
+	// The ID of the job
+	ID int `toml:"id"`
+	// The threshold time that the job will use internally to do a task
+	Threshold string `toml:"threshold"`
+	// How often to run the job
+	Every string `toml:"every"`
+	// Is the job running
+	Running bool `toml:"running"`
+}
+
+func OldConfigOrDefault(filename string) (*OldJobConfig, error) {
+	return readOldJobConfigFromFile(filename)
+}
+
+type OldJobConfig struct {
+	jobsConfig []JobConfig `toml:"jobs_config"`
+}
+
+func (c *OldJobConfig) ConfigForJob(jobIndex int) (JobConfig, error) {
+	if jobIndex < len(c.jobsConfig) {
+		return c.jobsConfig[jobIndex], nil
+	} else if jobIndex < len(defaultJobConfig) {
+		return defaultJobConfig[jobIndex], nil
+	}
+	return JobConfig{}, errors.New("no default configuration for job")
+}
+
+var defaultJobConfig = []JobConfig{
+	DeleteNodes: JobConfig{
+		ID:        DeleteNodes,
+		Threshold: "1d",
+		Every:     "15m",
+		Running:   false,
+	},
+	NodesMissing: JobConfig{
+		ID:        NodesMissing,
+		Threshold: "1d",
+		Every:     "15m",
+		Running:   true,
+	},
+	MissingNodesForDeletion: JobConfig{
+		ID:        MissingNodesForDeletion,
+		Threshold: "30d",
+		Every:     "15m",
+		Running:   true,
+	},
+}
+
+func readOldJobConfigFromFile(configFile string) (*OldJobConfig, error) {
```

Comment:
kinda strange that this doesn't return an error if the file doesn't exist. I would expect it to return the error and it would be up to the caller to know whether or not they wish to proceed if the file doesn't exist.